doc/get_started/win-10-iot-ent-ltsb-sa-3000p-c.md

```diff
@@ -0,0 +1 @@
+---
platform: windows 10 iot enterprise lstb
device: sa-3000p
language: c
---

Run a simple C sample on SA-3000P device running Windows 10 IoT Enterprise LTSB
===
---

# Table of Contents

-   [Introduction](#Introduction)
-   [Step 1: Prerequisites](#Prerequisites)
-   [Step 2: Prepare your Device](#PrepareDevice)
-   [Step 3: Build and Run the Sample](#Build)


<a name="Introduction"/>
# Introduction

**About this document**

This document describes how to connect SA-3000P device running Windows 10 IoT Enterprise LTSB with Azure IoT SDK. This multi-step process includes:

-   Configuring Azure IoT Hub
-   Registering your IoT device
-   Build and deploy Azure IoT SDK on device

<a name="Prerequisites"></a>
# Step 1: Prerequisites

You should have the following items ready before beginning the process:

-   [Prepare your development environment][setup-devbox-windows]
-   [Setup your IoT hub][lnk-setup-iot-hub]
-   [Provision your device and get its credentials][lnk-manage-iot-hub]
-   SA-3000P device.

<a name="PrepareDevice"></a>
# Step 2: Prepare your Device

-   Turn on your SA-3000P device and connect to internet

<a name="Build"></a>
# Step 3: Build SDK and Run the sample

-   Start a new instance of Visual Studio 2015. Open the **azure_iot_sdks.sln** solution in the **cmake** folder in your home directory.

-   In Visual Studio, in **Solution Explorer**, navigate to **simplesample_amqp** project, open the **simplesample_amqp.c** file.

-   Locate the following code in the file:

        static const char* connectionString = "HostName=medicore-iothub.azure-devices.net;DeviceId=MyNewDevice;...";

-   Replace the above placeholder with device connection string you obtained in [Step 1](#Step-1:-Prerequisites) and save the changes.

-   See [Manage IoT Hub][lnk-manage-iot-hub] to learn how to observe the messages IoT Hub receives from the application.

-   In **Solution Explorer**, right-click the **simplesample_amqp** project, click **Debug**, and then click **Start new instance** to build and run the sample. The console displays messages as the application sends device-to-cloud messages to IoT Hub.

-   See [Manage IoT Hub][lnk-manage-iot-hub] to learn how to send cloud-to-device messages to the application.

[setup-devbox-windows]: https://github.com/Azure/azure-iot-sdks/blob/master/c/doc/devbox_setup.md
[lnk-setup-iot-hub]: ../setup_iothub.md
[lnk-manage-iot-hub]: ../manage_iot_hub.md













































```

Comment:
Ok. I see it in CodeFlow, but not in GitHub for some reason.

---
In reply to: [85573225](https://github.com/Azure/azure-iot-sdks/pull/890#discussion_r85573225) [](ancestors = 85573225)